Kernel/Net/Realtek/RTL8139NetworkAdapter.h

```diff
@@ -32,6 +32,7 @@ class RTL8139NetworkAdapter final : public NetworkAdapter
     virtual bool link_full_duplex() override;
 
     virtual StringView purpose() const override { return class_name(); }
+    virtual StringView device_name() const override { return "RTL8139"sv; }
```

Comment:
You add this implementation here, but it seems like it's not actually used? Because I don't see any of the logging function updated to the use the new API. Same for the similar members. I think this could be used at many more places now?

Reply:
Yep - once all are happy with the signature of `dmesgln_pci` I will go refactor all of the `PCI::Devices` to use the new API.

Reply:
Would probably be nice to do that as a separate commit in this PR, because only the having the current commit will basically add 'dead code' to the project, which is unlikely to be merged.

Reply:
I've gone through and refactored all of the `PCI::Device` users of `dmesgln` to now use `dmesgln_pci`. I propagated errors with `TRY` where possible, but had to use `release_value_but_fixme_should_propagate_errors()` at some call sites.  What do you think about these changes?